prereise/gather/demanddata/transportation_electrification/const.py

```diff
@@ -1,5 +1,53 @@
 model_year = 2040
 
+ldv_location_allowed = {
+    1: {1},  # home only
+    2: set(range(1, 13)),  # home and go to work related
+    4: {1, 21},  # home and school only
+    5: {1, 11, 12, 21},  # home and work and school
+    6: {10, 11, 12},  # only work
+}
+
+hdv_location_allowed = {
+    1: {1},  # base only
+}
+
+why_to_list = [
+    1,
+    53,  # home related
+    10,
+    11,
+    12,  # work related
+    13,
+    14,
+    17,
+    20,
+    21,
+    22,
+    23,  # other
+    24,
+    30,
+    40,
+    41,
+    42,
+    50,
+    51,
+    52,
+    53,
+    54,
+    55,
+    60,
+    61,
+    62,
+    63,
+    64,
+    65,
+    80,
+    81,
+    82,
+    83,
+]
```

Comment:
`53` is duplicated. What is the reason for that?

Reply:
The extra `53` appears to just be a typo.  
Here's the original list of the categories, pages 38-39:
https://nhts.ornl.gov/assets/NHTS2017_UsersGuide_04232019_1.pdf

This long list of codes is from the 2009 NHTS dataset.  The 2017 codes are much simpler.